### PR TITLE
FOGL-2988 Add packaging files

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
Please add requirements for deb and rpm packager in Package file And add `foglamp.version` file as required for packaging.

**Comment by ashish-jabble** [comment]
Also it is failing on ubuntu16.04 (- Configuring incomplete, errors occurred!). Below are the attached logs
[dnp3.txt](https://github.com/foglamp/foglamp-south-dnp3/files/3368689/dnp3.txt)
It may need require changes in requirements.sh

**Comment by MarkRiddoch** [comment]
> Also it is failing on ubuntu16.04 (- Configuring incomplete, errors occurred!). Below are the attached logs
> [dnp3.txt](https://github.com/foglamp/foglamp-south-dnp3/files/3368689/dnp3.txt)
> It may need require changes in requirements.sh

I think we may be best to put the Ubuntu 16 issue in a separate Jira and get Massimiliano to look into it when he gets back.

**Comment by ashish-jabble** [comment]
> I think we may be best to put the Ubuntu 16 issue in a separate Jira

Actually it is giving error on all the platforms. Should we exclude this plugin from CI packaging jobs for now or let them fail on CI?

**Comment by MarkRiddoch** [comment]
Pushed some more changes that should now resolve this on Ubuntu 16 and 18. Also Raspbian, but not yet been able to test CentOS/RedHat platforms.

File: Package [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+case "$arch" in
+	x84_64)
+		;;
+	armv7l)
+		;;
+	aarch64)
+		;;
+esac
+case "$package_manager" in
+	deb)
```
Comment (ashish-jabble) [inline Package:22]: Please add requirements which are needed for this plugin.

**Comment by ashish-jabble** [comment]
@MarkRiddoch Except ub16, all fine on ub18 and Rpi.

**Comment by MarkRiddoch** [comment]
The opendnp3 library does not compile on CentOS 7 currently. Therefore let's exclude this plugin from CentOS/RHEL for the time. A new Jira will be opened for support on that platform.

Could you tell me what you are seeing on Ubuntu 16 please? I see the package being generated without any reported problems.

**Review by ashish-jabble — APPROVED** [review]
It seems like something with my environment. Now all good.

**Pull request merged**